lib/src/excel.dart

```diff
@@ -126,6 +175,44 @@ class ExcelDart {
     }));
   }
 
+  /// Highlight rows that are underpacing.
+  ///
+  /// Structured reference (i.e. Table[@Column]) is not allowed in Excel
+  /// conditional formatting. Regular reference (i.e. $A5) has to be used
+  /// instead. As table expands, Excel would automatically extends the
+  /// conditional formatting rule to cover the entire range.
+  static void _highlightUnderpacingRows(String tableName) async {
+    await ExcelJS.run(allowInterop((context) async {
+      final table = context.workbook.tables.getItem(tableName);
+      final range = table.getDataBodyRange();
+      final format = range.conditionalFormats.add('Custom');
+
+      // Excel formula DATEDIF(dateA, dateB, "D") calculates the difference
+      // between two dates in days. This formula calculates:
+      // (spent / budget) * (flight duration/ current duration)
+      // if current duration or flight duration is zero days, then the formula
+      // simply calculates (spent / budget).
+      final formula = '''
+      IF(OR(DATEDIF($_startDateColumn, $_endDateColumn, "D") = 0, 
+            DATEDIF($_startDateColumn, NOW(), "D") = 0),
+         ($_spentColumn/$_budgetColumn),   
+         ($_spentColumn/$_budgetColumn) * 
+         (
+           DATEDIF($_startDateColumn, $_endDateColumn, "D") /
```

Comment:
same question re:division. Is it integer division?

Reply:
Ugh...sorry, forgot to include this in the comments. Nope, not integer division, just Excel regular division.